docs/docs/queries/structure.md

```diff
@@ -1,44 +1,207 @@
+
+It behaves differently from the other Data Commands: You can add **zero or one** `FROM` data command to your query, right after your Query Type. You cannot add multiple FROM statements and you cannot add it after other Data Commands.
+
+~~~
+Lists all pages inside the folder Books and its sub folders
+```dataview
+LIST
+FROM "Books"
+```
+
+Lists all pages that include the tag #status/open or #status/wip
+```dataview
+LIST
+FROM #status/open OR #status/wip
+```
+
+Lists all pages that have either the tag #assignment and are inside folder "30 School" (or its sub folders), or are inside folder "30 School/32 Homeworks" and are linked on the page School Dashboard Current To Dos
+```dataview
+LIST
+FROM (#assignment AND "30 School") OR ("30 School/32 Homeworks" AND outgoing([[School Dashboard Current To Dos]]))
+```
+
+~~~
+
+!!! info "Read more about `FROM` [here](./data-commands.md#from)."
+
+## Filter, sort, group or limit results
+
+In addition to the Query Types and the **Data command** `FROM` that's explained above, you have several other **Data Commands** available that help you restrict, refine, sort or group your query results. 
+
+All data commands except the `FROM` command can be used **multiple times in any order** (as long as they come after the Query Type and `FROM`, if `FROM` is used at all). They'll be excuted in the order they are written.
+
+Available are:
+
+1. **FROM** like explained [above](#choose-your-source).
+2. **WHERE**: Filter notes based on information **inside** notes, the meta data fields.
+3. **SORT**: Sorts your results depending on a field and a direction.
+4. **GROUP BY**: Bundles up several results into one result row per group.
+5. **LIMIT**: Limits the result count of your query to the given number.
+6. **FLATTEN**: Splits up one result into multiple results based on a field or calculation.
+
+~~~
+
+Lists all pages that have a metadata field `due` and where `due` is before today
+```dataview
+LIST
+WHERE due AND due < date(today)
+```
+
+Lists the 10 most recently created pages in your vault that have the tag #status/open
+```dataview
+LIST
+FROM #status/open
+SORT file.ctime DESC
+LIMIT 10
 ```
+
+Lists the 10 oldest and incompleted tasks of your vault as an interactive task list, grouped by their containing file and sorted from oldest to newest affected file.
+```dataview
+TASK
+WHERE !completed
+SORT created ASC
+LIMIT 10
+GROUP BY file.link
+SORT rows.file.ctime ASC
+```
+
 ~~~
 
-Only the [**Query Type**](./query-types.md) (table/list/task/calendar) statement is required - if the "from" statement is omitted, the query runs for all files in
-your vault. You can specify [data commands](./data-commands.md) like `WHERE` multiple times; they will run in the order they are written. 
+!!! info "Find out more about available [data commands](./data-commands.md)."
+
+## Examples
+
+Following are some examples of valid query structures. Find more examples [here](../resources/examples.md).
```

Comment:
Can be simplified to "following are some valid example queries". Valid isn't entirely necessary either, it's safe to assume examples in the docs work, or at least that's what most folks would expect.